airbyte-integrations/connectors/source-rki-covid/source_rki_covid/schemas/german_history_deaths.json

```diff
@@ -0,0 +1 @@
+{"$schema": "http://json-schema.org/schema#", "type": "object", "properties": {"deaths": {"type": "integer"}, "date": {"type": "string"}}, "required": ["date", "deaths"]}
```

Comment:
please run format to format those files.

Reply:
Formatted.